src/Microsoft.Dnx.DesignTimeHost/Models/OutgoingMessages/DependencyDescription.cs

```diff
@@ -22,6 +24,31 @@ public class DependencyDescription
 
         public IEnumerable<DependencyItem> Dependencies { get; set; }
 
+        public DiagnosticMessage Error { get; set; }
```

Comment:
Why Error and warning are not collections? It is not possible in the future to have several errors r warnings for one dependency?  Or we will change it later if needed?